### DISCUSSION TITLE
DeepMD for non-potential force fields

### DISCUSSION THREAD

**Discussion by kolotinsky1998** [deepmodeling/deepmd-kit#2617] [General]
Dear DeepMD community!

I am trying to use DeepMD for predicting forces in a system with non-reciprocal interactions. A simple example of such a system is a two-particle system, where the force from the first particle to the second particle is not equal to the minus force from the second particle to the first one, in other words, the third Newton's law is broken in the system. 

My several experiments trying to describe forces of such a system with non-reciprocal interaction using DeepMD were unsuccessful. I am wondering is that task conceptually possible with DeepMD descriptors or not?

Best regards,
Daniil Kolotinskii

**Comment by njzjz** [comment]
The energy model represents forces as the negative gradient of the potential energy, which means the forces are [conservative](https://en.wikipedia.org/wiki/Conservative_force).
In your case, I guess you need to use a tensorial model to directly fit non-conservative forces.

**Comment by kolotinsky1998** [comment]
Thank you for the fast answer! Is there such a tensorial model in DeepMD? If yes, what is the name of descriptor for it?

**Comment by njzjz** [comment]
Please see https://docs.deepmodeling.com/projects/deepmd/en/master/model/train-fitting-tensor.html. The keyword is `dipole` although the fitting data may not be actually dipole.

**Comment by kolotinsky1998** [comment]
Thanks a lot, will check it!
